libcontainer/container_linux.go

```diff
@@ -836,6 +836,84 @@ func (c *linuxContainer) handleCriuConfigurationFile(rpcOpts *criurpc.CriuOpts)
 	}
 }
 
+func (c *linuxContainer) handleCheckpointingExternalNamespaces(rpcOpts *criurpc.CriuOpts, t configs.NamespaceType) error {
+	var nsType string
+	var minVersion int
+	switch configs.NEWNET {
```

Comment:
whoops, you probably mean to write `switch t` here.

Reply:
fixed